src/election_anomaly/__init__.py

```diff
@@ -123,6 +123,7 @@ def track_results(self, shortname, election):
 
         if error != []:
```

Comment:
An empty python list (and dictionary) is `None`. So you can test this by saying `if error:`.